README.md

```diff
@@ -1,24 +1,101 @@
 # Validations in Controller Actions
 
-## Objectives
+Now that we know Rails automatically performs validations defined on models,
+let's use this information to easily display validation errors to the user.
 
-  - Define validations on a model
-  - Use the validation state of a model in a response conditional in an action
-  - re-render a form with validation errors
-  - validate a create action
-  - validate an update action
+At this point, we'll be covering step two of the following flow:
 
-## Notes
+1. User fills out the form and hits "Submit", transmitting the form data via
+   a POST request.
+2. **The controller sees that validations have failed, and re-renders the
+   form.**
+3. The view displays the errors to the user.
 
-Really simple readme, given a form and a model with validations, what does a controller action that takes this validation into account look like?
 
-We need to see if the object is valid on form submission, if so, save it, and redirect to the newly created resource.
+## A Note About Page Loads
 
-if not, then we want to re-render the form with the invalid instance in scope so that we can use it to introspect on erorrs (more on that very soon, but we need to know what was wrong.) its important to try to remind the student that the @post from Post#new is long gone - every request happens indepedntly, we've recreated the essence of @post from Post#new when the form was submitted with Post.new(params[:post]). We can't redirect to new to reshow the form because then the invalid @post based on params[:post] is gone. Try to explain this idea. Why we render on invalid objects instead of redirecting.
+You're probably used to validations happening almost instantaneously on websites
```

Comment:
I'd probably stay away from mentioning JavaScript in here. The important thing to get across is that each request happens independently from any other request, so variables/state/etc from one request are gone on the next request. I'd even consider just removing this section entirely, and moving the discussion of request independence down to where you're showing how to check the validation in the controller.